UnitTests/fixtures/LocStructuredOnly.strings

```diff
@@ -0,0 +1,3 @@
+"settings.navigation-bar.title.even.deeper" = "Settings";
+"SETTINGS.USER_PROFILE_SECTION.HEADER_TITLE" = "User Profile Settings";
+"seTTings.uSer-proFile-sectioN.footer_text" = "Here you can change some user profile settings.";
```

Comment:
As you can see in `Strings-File-Structured.swift.out` the result looks like this:

```
case Settings(SettingsL10n)
enum SettingsL10n {
```

As you can see there is also a **case** names `Settings` so if there was also a leaf names as such then this would lead to two entries techincally rendering the result as invalid code. If we want nodes to be autocompleting with the same name I think there's no way to fix this so long as we use enums. We could of course choose one of the two – but that would lead to confusions on the user side so I think it's best to actually keep it the way it is and document well. Long term we can fix this by using one of the other approaches.